src/Compilers/Core/Portable/Operations/OperationConstantValue.cs

```diff
@@ -0,0 +1,95 @@
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#nullable enable
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Root type for operation constant values.
+    /// </summary>
+    internal abstract class OperationConstantValue
```

Comment:
>OperationConstantValue [](start = 28, length = 22)

It feels like creating a wrapper is unnecessary, simply holding onto ConstantValue instance should be sufficient.